### PR TITLE
[WNMGDS-450] [WMNGDS-441] [WMNGDS-451] Sketch file updates

### LINKED ISSUE #339: Sketch file: improve naming and organization of symbols and styles
Related to https://github.com/CMSgov/design-system/issues/331

Improve naming and organization of symbols, text styles, and layer styles - ensure naming matches what's used in the documentation, revisit submenu organization.

### PR DESCRIPTION
**Pull request by line47**
### Changed
Made the names of layers in the Sketch file more representative of the symbol or layer being used. 

### Removed
Removed "other"layer styles from the Sketch file that were unused or unnecessary 

### Fixed
Fixed checkbox and radio buttons from shifting when selecting different states of default, focused, checked and unchecked

Closes #339 

**How to test**

[Get the latest Sketch file here ](https://github.com/CMSgov/design-system/blob/WNMGDS-450-Sketch-file-update/design-assets/CMS-Design-System-UI-kit.sketch)

### CODE REVIEW

**Comment by kathleenmeditz** [comment]
The radio buttons & checkboxes look good for default, selected, and unselected states. **I'm not seeing the focused state in my UI kit** (I just re-downloaded it from design.cms.gov and added it as a library, so I'm assuming I have the most up-to-date version but I could be wrong). Any idea why I might be missing the focus state in my library?


Other than that, there are some **small inconsistencies with naming conventions** that I've listed below:
- The `Desktop - Display Text` should be updated to `Desktop - Display Style` for consistency with the title and mobile display text naming conventions.

- To be consistent with the naming conventions for other link text styles, the `Visited Link` name should be changed to `Link - Visited`.

- `Copy - Small` should be changed to `Body Copy - Small` to be consistent with the naming convention for body copy. 

- Similarly, `Body - Single Line` can be changed to `Body Copy - Single Line` and `Body, Bold, Single line` can be changed to `Body Copy - Bold Single Line`.

- `Body, Bold, Single line Right Align` can be changed to `Body Copy - Bold Single Line, Right Align` 

- `Body - Single line, Right align` can be changed to `Body Copy - Single Line, Right Align`

- `Tab Panel` can be `Tab - Panel`

- Add `Component` in front of `Autocomplete / Unselected Option`, `Autocomplete / Selected Option, ` Autocomplete / Loading`

**Comment by line47** [comment]
@kathleenmeditz please use the Sketch file in the PR to test (you can download this) 
https://github.com/CMSgov/design-system/blob/WNMGDS-450-Sketch-file-update/design-assets/CMS-Design-System-UI-kit.sketch

**Comment by kathleenmeditz** [comment]
TLDR; There are some naming convention fixes needed, but I'm not sure if those fall into the scope of this PR. I'm documenting them here just in case, but it might be worth opening up a new PR specifically for addressing naming conventions. Other than these, looks good!

- "Default alerts" are missing the correct naming convention (e.g.,  `Alert info` should be changed to `Component / Alert / Default (?) / info`.
- "Tab" and "Help Drawer" components should also be updated to follow component naming conventions 
- "Spinner" icons should be updated to follow icon naming conventions
- The medium sized badges are all named `... / normal Copy`, so we should consider what we are calling this size badge. Maybe the circular badges are `small`, and medium sized badges are `normal`.
-  The "Date field" layers should be grouped to form their own, distinct `Date field` component

**Inverse components:**
- Consider adding inverse to the name of all inverse components. Currently they are copies of default components (e.g., `Component / Button / Inverse / hover` ). 
- Same goes for the hover, active, focus, and disabled Inverse transparent buttons, inverse checkboxes, inverse radio buttons, and inverse form controls.
- I understand some of these have an inverse text style applied, but specifically naming the inverse component could make it easier to add into a Sketch file straight form the library instead of needing to update the text style of the default component.

**Comment by line47** [comment]
I'm pulling this these items out into new tickets

**Ticket one**
- The "Date field" layers should be grouped to form their own, distinct Date field component

**Ticket two**
Inverse components:

- Consider adding inverse to the name of all inverse components. Currently they are copies of default components (e.g., Component / Button / Inverse / hover ).
- Same goes for the hover, active, focus, and disabled Inverse transparent buttons, inverse checkboxes, inverse radio buttons, and inverse form controls.
- I understand some of these have an inverse text style applied, but specifically naming the inverse component could make it easier to add into a Sketch file straight form the library instead of needing to update the text style of the default component.

**Comment by line47** [comment]
@kathleenmeditz let me know if you can re-review or not 👍

**Comment by kathleenmeditz** [comment]
Aside from the two tickets mentioned (date field, inverse components), I think it could be helpful to also have a naming convention ticket  that addresses the tab, help drawer, spinner icon, and badge components. Getting these updated should make them easier to find when adding them via the Sketch library function. Otherwise, looks good @line47

**Comment by line47** [comment]
I updated the tab, help drawer, spinner icon, and badge components in that PR - if you download the Sketch file again you should be able to check and make sure those items are addressed.

**Comment by kathleenmeditz** [comment]
Nice! Reviewed the Sketch file, looks great 👍
